source/client/flush_worker_impl.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
+
 // Flush worker implementation. Flush worker periodically flushes metrics
 // snapshot to all configured stats sinks in Nighthawk.
 #pragma once
```

Comment:
Please remove.

Reply:
Done in 78338dca6de640193a4ea395ed2052c832397ea8.